internal/rukpak/convert/registryv1_test.go

```diff
@@ -559,10 +584,47 @@ func convertToUnstructured(t *testing.T, obj interface{}) unstructured.Unstructu
 func findObjectByName(name string, result []client.Object) client.Object {
 	for _, o := range result {
 		// Since this is a controlled env, comparing only the names is sufficient for now.
-		// In future, compare GVKs too by ensuring its set on the unstructuredObj.
+		// In the future, compare GVKs too by ensuring its set on the unstructuredObj.
 		if o.GetName() == name {
 			return o
 		}
 	}
 	return nil
 }
+
+func newBundleFS() fstest.MapFS {
+	annotationsYml := `
+annotations:
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.package.v1: test
+`
+
+	csvYml := `
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: test.v1.0.0
+  annotations:
+    olm.properties: '[{"type":"from-csv-annotations-key", "value":"from-csv-annotations-value"}]'
+spec:
+  installModes:
+    - type: AllNamespaces
+      supported: true
```

Comment:
I can see here that we still using the same mock from what was removed `csv.Spec.InstallModes = []v1alpha1.InstallMode{{Type: v1alpha1.InstallModeTypeMultiNamespace, Supported: true}}`